CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing
+
+## Adding a new rule
+New rules can be implemented in two ways:
```

Comment:
I propose adding a new line after the title to improve readability and for consistency with other markdown files:
```suggestion
## Adding a new rule

New rules can be implemented in two ways:
```